app/services/ethiopia_calendar_utilities.rb

```diff
@@ -6,4 +6,8 @@ def self.ethiopian_to_gregorian(e_year, e_month, e_days)
 
     Date.new(year, 1, 1) + days_offset.days
   end
+
+  def self.gregorian_month_period_to_ethiopian(period)
+    period.advance(years: -7, months: -8)
```

Comment:
Is this correct? Shouldn't it be (more or less) the inverse of `ethiopian_to_gregorian`? Is it ok that this is just more of a ballpark estimate?

Reply:
I believe the ethiopia ministry of health shoehorns ethiopia dates into their DHIS2 instance using this ballpark method, hence it's a lot simpler than a real conversion.

Should we qualify this method name to indicate that more clearly? Perhaps with some sort of `dhis2` prefix or suffix?